src/core/Akka.Remote.Tests/Transport/AkkaProtocolSpec.cs

```diff
@@ -181,7 +180,7 @@ public async Task ProtocolStateActor_must_in_inbound_mode_accept_payload_after_A
 
             wrappedHandle.ReadHandlerSource.SetResult(new ActorHandleEventListener(TestActor));
 
-            Assert.True(collaborators.FailureDetector.called);
+            Assert.True(collaborators.FailureDetector.IsMonitoring);
```

Comment:
Need to change this to the same assertion if the last change proves to fix the racy condition